modules/migration-using-cpma.adoc

```diff
@@ -3,21 +3,26 @@
 [id='migration-using-cpma_{context}']
 = Using CPMA
 
+CPMA can run in remote mode, retrieving the configuration files using SSH, or in local mode, using local copies of the configuration files.
+
 .Prerequisites
 
-The following prerequisites apply to the {product-title} 3 cluster:
+The following prerequisites apply:
 
-* The cluster is updated to the latest synchronous release.
-* The environment health check has been run to confirm that there are no diagnostic errors or warnings.
-* The {product-title} version is 3.7, 3.9, 3.10, or 3.11.
+* The {product-title} 3 cluster must be updated to the latest synchronous release.
+* The environment health check must be run on the {product-title} 3 cluster to confirm that there are no diagnostic errors or warnings.
+* The {product-title} version must be 3.7, 3.9, 3.10, or 3.11.
+* If you are running CPMA in remote mode, the SSH user must have sudo permissions to read the configuration files. If this is not possible, you must copy the files locally and run CPMA in local mode.
+* You must have `cluster-admin` privileges.
```

Comment:
What about proving the grant command too?
`oc adm policy add-cluster-role-to-user cluster-admin <username>`

Reply:
I didn't add this line because I didn't see it as relevant to this procedure. If someone needs to grant cluster-admin privileges to the user, it is well documented elsewhere.

Reply:
Fair enough but what about adding a link to the documentation then?

Reply:
OK, will do.